client/app/components/BigMessage.jsx

```diff
@@ -1,19 +1,21 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { uniqueId } from "lodash";
 
 function BigMessage({ message, icon, children, className }) {
+  const messageId = uniqueId("bm-message");
```

Comment:
This partially solves the thing as it will execute `uniqueId` regardless. Problem with this is that iirc, Lodash uses some sort of global var to track and sum up the ids. (We would end up with some possibly big ones 😝)

Reply:
In brief, that was to avoid [this](https://reactjs.org/docs/hooks-faq.html#how-to-create-expensive-objects-lazily) as that will avoid the component id to be updated, but not the Lodash reference to the "next id"